### PR TITLE
Style: Handle default kwargs

### PR DESCRIPTION
**Pull request by adrien-berchet**

### CODE REVIEW

File: tmd/view/view.py [hunk 1]
```diff
@@ -327,8 +328,9 @@ def neuron(
     to_plot = []
 
     if rotation == "apical_dendrite":
-        angle = np.arctan2(nrn.apical_dendrite[0].get_pca()[0], nrn.apical_dendrite[0].get_pca()[1])
-        angle = np.arctan2(rotation[1], rotation[0])
+        rotation = np.arctan2(
+            nrn.apical_dendrite[0].get_pca()[0], nrn.apical_dendrite[0].get_pca()[1]
+        )
```
Comment (adrien-berchet) [inline tmd/view/view.py:333]: These lines were very weird, I am not sure how the 'rotation' argument is supposed to be used.

File: examples/Advanced/advanced_plot.py [hunk 2]
```diff
@@ -173,11 +176,14 @@ def barcode_mirror(ph, new_fig=True, subplot=False, color="b", **kwargs):
         if p[2] < 0:
             ax.plot(_np.subtract([0, 0], p[:2]), [ip, ip], c=color)
 
-    kwargs["title"] = kwargs.get("title", "Mirror Barcode of p.h.")
-    kwargs["xlabel"] = kwargs.get("xlabel", "Lifetime")
+    all_kwargs = {
+        "title": "Mirror Barcode of p.h.",
```
Comment (lidakanari) [inline examples/Advanced/advanced_plot.py:180]: Maybe change to "Mirror persistence barcode"

File: tmd/view/plot.py [hunk 3]
```diff
@@ -330,10 +351,13 @@ def start_length_diagram(ph, new_fig=True, subplot=False, color="b", alpha=1.0,
     for p in ph_transformed:
         ax.scatter(p[0], p[1], c=color, edgecolors="black", alpha=alpha)
 
-    kwargs["title"] = kwargs.get("title", "Transformed Persistence diagram")
-    kwargs["xlabel"] = kwargs.get("xlabel", "Start of the component")
-    kwargs["ylabel"] = kwargs.get("ylabel", "Length of the component")
-    return cm.plot_style(fig=fig, ax=ax, **kwargs)
+    all_kwargs = {
+        "title": "Transformed Persistence diagram",
```
Comment (lidakanari) [inline tmd/view/plot.py:355]: Transformed persistence diagram

File: tmd/view/view.py (hunk 1 above)
Comment (lidakanari) [reply tmd/view/view.py:333]: Maybe modify to "apical_alignment" and allow True / False

File: tmd/view/view.py [hunk 4]
```diff
@@ -246,69 +247,64 @@ def neuron(
     hadd=0.0,
     vadd=0.0,
     neurite_type="all",
-    rotation=None,
-    nosoma=False,
+    apical_alignment=False,
+    plot_soma=True,
```
Comment (adrien-berchet) [inline tmd/view/view.py:251]: I also renamed this one, is it ok?

File: tmd/view/view.py [hunk 5]
```diff
@@ -318,17 +314,18 @@ def neuron(
...
         soma(nrn.soma, plane=plane, hadd=hadd, vadd=vadd, **kwargs)
 
     h = []
     v = []
 
     to_plot = []
 
-    if rotation == "apical_dendrite":
+    if apical_alignment:
         angle = np.arctan2(nrn.apical_dendrite[0].get_pca()[0], nrn.apical_dendrite[0].get_pca()[1])
```
Comment (lidakanari) [inline tmd/view/view.py:326]: I suspect this will fail if there is no apical tree. Could we raise if neuron has no apical?

Comment (adrien-berchet) [reply tmd/view/view.py:330]: Indeed, thanks!
I rebased on master and fixed this.

**Review by lidakanari — COMMENTED** [review]
Lgtm. One small comment, I think you did not modify the kwargs for the function "density_cloud". Maybe it should also change for homogeneity. If you change this function, it also has an error: line 1066 (new) should be alpha, it is now alhpa. I can fix later if this is not modified.

**Comment by adrien-berchet** [comment]
I didn't see the `alhpa` indeed, I can fix it in this PR.

For `density_cloud` there is no default value in this case, we just force 2 values, so I didn't change anything. But we could use `update` to force values (several other functions should also be modified then, like `all_trunks3d` and `population3d` for example), should I?

**Pull request merged**